### ISSUE TITLE
"flowtype is not a function"

### ISSUE THREAD

**Issue by bobpaul** [imnotbob/EnvoyHE#2]
The graph app hasn't been working for me. I finally looked in the javascript console and I see "Uncaught TypeError: $(...).flowtype is not a function" around this block of code in `deviceTiles`. 

```
<script>
	$("body").flowtype({
		minFont: 7,
		maxFont: 10,
		fontRatio: 30
	});
</script>
```

There's also an error that loading failed for `https://fastcdn.org/FlowType.JS/1.1/flowtype.js`. That file fails to load in Chrome, Firefox, and even using Curl. It seems `fastcdn.org` is broken. While looking for an alternative source of the file, I [found a notice](https://github.com/simplefocus/FlowType.JS) that flowtypes is no longer necessary as CSS is now capable of whatever functionality flowtypes provided.

**Comment by imnotbob** [comment]
So have you created a fix?

**Comment by bobpaul** [comment]
no, I tried using `https://raw.githubusercontent.com/simplefocus/FlowType.JS/master/flowtype.js` from that other repo, but github returns `text/plain` as the mime type and browsers reject that for javascript.

**Comment by imnotbob** [comment]
Found this is part of envoy...

**Comment by bobpaul** [comment]
I thought it was [SolarGraphs.groovy:842](https://github.com/imnotbob/EnvoyHE/blob/6e6e4c1a9e595c0b63bced74bc4eb08f7774bbc3/app/solarGraphs.groovy#L842). Is it also somewhere else?

**Comment by imnotbob** [comment]
I pushed an update for this
